libsel4/arch_include/arm/interfaces/sel4arch.xml

```diff
@@ -23,7 +23,7 @@
             <param dir="in" name="vaddr" type="seL4_Word"
             description="Virtual address to map the page into."/>
             <param dir="in" name="attr" type="seL4_ARM_VMAttributes">
-                <description> 
+                <description>
                     VM Attributes for the mapping. <docref>Possible values for this type are given in <autoref label="ch:vspace"/>  .</docref>
```

Comment:
Could you move these space changes to a dedicated commit?

Reply:
Yeah, I can do that.